conbench/templates/run.html

```diff
@@ -51,6 +52,7 @@
                 {% endfor %}
             </tbody>
         </table>
+        {{ wtf.quick_form(form, id="run-form", button_map={'delete': 'primary'}) }}
```

Comment:
This button should only show up if you're authenticated, right now it shows up either way. The API layer won't actually let you delete it if you're unauthenticated, but it should still be hidden.

For benchmark delete, this was accomplished like so:

```
class Benchmark(AppEndpoint, BenchmarkMixin, RunMixin, TimeSeriesPlotMixin):
    def page(self, benchmark, run, form):
        if not flask_login.current_user.is_authenticated:
            delattr(form, "delete")
```

I suspect something similar can be done for deleting runs.

Reply:
Thanks, changed